improver_tests/cube_combiner/test_CubeMultiplier.py

```diff
@@ -75,7 +75,6 @@ def test_vicinity_names(self):
         cube.data = np.ones_like(cube.data)
         cube.remove_coord("lwe_thickness_of_precipitation_amount")
         cubelist = iris.cube.CubeList([self.cube4.copy(), cube])
-        input_copy = deepcopy(cubelist)
```

Comment:
Was this line removed accidentally?

Reply:
The `input_copy` from line 78 doesn't seem to be used for the rest of this test function, so looks safe to remove. I suspect that this line was copy-pasted from the previous function, where does get used as part of the test:
 https://github.com/metoppv/improver/blob/177a0343f034404e22400e0305d126aa386a5653/improver_tests/cube_combiner/test_CubeMultiplier.py#L58

https://github.com/metoppv/improver/blob/177a0343f034404e22400e0305d126aa386a5653/improver_tests/cube_combiner/test_CubeMultiplier.py#L66

Reply:
Yep, sorry @dmentipl - yes @tjtg, that was what I think happened.